bento/src/main/java/com/yelp/android/bento/core/ComponentGroup.java

```diff
@@ -40,6 +44,9 @@ public ComponentGroup() {
         mSpanSizeLookup = new SpanSizeLookup() {
             @Override
             public int getSpanSize(int position) {
+                if (hasGap(position)) {
```

Comment:
This looks like a bug fix for something. Is it related to this PR or should we make it separate?

Reply:
Eh, back when we were testing `ComponentGroups` with gaps, it crashed and this fixed it. Now that we eliminated that feature, the test app works. I will open a separate PR for this.